functions/Get-DbaSpConfigure.ps1

```diff
@@ -55,55 +55,58 @@ function Get-DbaSpConfigure {
 		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
 		[Alias("ServerInstance", "SqlServer", "SqlServers")]
 		[DbaInstanceParameter[]]$SqlInstance,
-		[System.Management.Automation.PSCredential]$SqlCredential
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[object[]]$Config
```

Comment:
I think this needs to match the property you have to return: `ConfigName` and not just `Config`. The object output for the value is set as `ConfigName`.